EKF/utils.cpp

```diff
@@ -58,3 +58,44 @@ matrix::Dcmf quatToInverseRotMat(const  matrix::Quatf &quat)
 
 	return dcm;
 }
+
+bool shouldUse321RotationSequence(const matrix::Dcmf& R) {
+	return fabsf(R(2, 0)) < fabsf(R(2, 1));
```

Comment:
Let's write the general comment here to explain the check; something like:
" We should use a 3-2-1 Tait-Bryan (yaw-pitch-roll) rotation sequence when there is more roll than pitch tilt and a 3-1-2 rotation sequence when there is more pitch than roll tilt to avoid gimbal lock."

So we can get rid of all the other similar comments